companies/joinclubhouse.json

```diff
@@ -10,14 +10,16 @@
     "runs": [
         "Clubhouse"
     ],
-    "address": "5335 Lawton Avenue\nOakland\nCalifornia 94618\nUnited States of America",
+    "address": "Alpha Exploration Co.\n548 Market St\nPMB 72878\nSan Francisco, California 94104-5401",
+    "phone": "415-549-1842",
```

Comment:
Not formatted correctly, `+1 415 549 1842`

Reply:
Do we have a source for this number? 🤔

Reply:
Negative, I think we should remove it. I doubt they would answer any of the phones listed anyway. 

I also found [this source](https://www.dnb.com/business-directory/company-profiles.alpha_exploration_co.2e9f2a57c1f5b0aebc20563d279ad536.html) with a different number.

Reply:
Thanks! I agree.